### DISCUSSION TITLE
Can I export the pdf in dark mode?

### DISCUSSION THREAD

**Discussion by 20112m** [sumatrapdfreader/sumatrapdf#3920] [Q&A]
I use inverted colours to view the pdf in dark mode and it works perfectly but I would like to save it as it is or export it in inverted colours.
Is this possible in Sumatra pdf please?

**Comment by GitHubRulesOK** [comment]
Inversion is just screen candy
the paper is always "white/light" otherwise the paper would be flooded wet and use a bottle of ink.

you can use many online tools to edit the source document but reversal of PDF colours is usually a generic failure.

In **some cases** Mutool invert (MuPDF is the core for SumatraPDF) can out put a reversal with all the problems that may entail.

**Reply by 20112m** [reply]
thank you so much for your answer :)
